frontend/packages/dev-console/src/components/project-access/__tests__/project-access-form-utils.spec.tsx

```diff
@@ -1,20 +0,0 @@
-import { filterRoleBindings, getUserRoleBindings } from '../project-access-form-utils';
-import { Roles } from '../project-access-form-utils-types';
-import {
-  roleBindingsResourceData,
-  roleBindingsWithRequiredRolesResult,
-  roleBindingsWithRequiredRoles,
-  roleBindingsWithRequiredAttributes,
-} from './project-access-form-data';
-
-describe('Fetch required roles', () => {
-  it('should fetch the only the required rolebindings', async () => {
-    const filteredRoleBindings = filterRoleBindings(roleBindingsResourceData, Roles);
-    expect(filteredRoleBindings).toEqual(roleBindingsWithRequiredRolesResult);
-  });
-
-  it('should consider only the required attributes', async () => {
-    const userRoleBindings = getUserRoleBindings(roleBindingsWithRequiredRoles);
-    expect(userRoleBindings).toEqual(roleBindingsWithRequiredAttributes);
-  });
-});
```

Comment:
Why remove this unit tests? The util functions still exist and are not changed, or? So I expect this tests would still run fine?

Reply:
fixed. Changed the file name to `project-access-form-utils.spec.ts`